src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--9.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity9;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity9() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity9);
+ 	 		Assert.assertTrue(enterQuantity9.isDisplayed());
+ 	 		enterQuantity9.clear();
+ 	 		enterQuantity9.sendKeys("2");
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--9.quantityAsDecimal']")
+ 	 //*[@id="change--9.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity10;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity10() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity10);
+ 	 		Assert.assertTrue(enterQuantity10.isDisplayed());
+ 	 		enterQuantity10.clear();
+ 	 		enterQuantity10.sendKeys("4");
+ 	 		}
+ 	 	
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--5.quantityAsDecimal']")
+ 	 //*[@id="change--5.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity7;
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity7() throws IOException {
+ 	 		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity7);
+ 	 		Assert.assertTrue(enterQuantity7.isDisplayed());
+ 	 		enterQuantity7.clear();
+ 	 		enterQuantity7.sendKeys("3");
+ 	 		}
+	
+	
+	
+ 	 	@FindBy(how = How.XPATH, using = "//a//following::span[text()='Update'][2]")
+ 		private WebElement clickUpdateButton;
+
+ 		/**
+ 		 * Method to click on Update button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 		public void clickUpdateButton() throws IOException {
+ 			CreateOrderPage sp = new CreateOrderPage(driver);
+ 			log.info("click on Update button.");
+ 			Assert.assertTrue(clickUpdateButton.isDisplayed());
+ 			clickUpdateButton.click();
+ 			JavaScriptExec.sleep();
+
+ 		
+
+	}
+	
+ 		@FindBy(how = How.XPATH, using = "//a//following::span[text()='Update'][2]")
+ 		private WebElement clickUpdateButton1;
+
+ 		/**
+ 		 * Method to click on Update button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 		public void clickUpdateButton1() throws IOException {
+ 			CreateOrderPage sp = new CreateOrderPage(driver);
+ 			log.info("click on Update button.");
+ 			Assert.assertTrue(clickUpdateButton.isDisplayed());
+ 			clickUpdateButton.click();
+ 			JavaScriptExec.sleep();
+
+ 		
+
```

Comment:
@mayankchhajer Please remove the extra lines.

Reply:
Done